extensions/http-curl/tests/VerifyInvokeHTTPTest.cpp

```diff
@@ -93,47 +95,55 @@ class VerifyInvokeHTTP : public CoapIntegrationBase {
     flowController_->unload();
     flowController_->stopC2();
 
-    runAssertions();
     cleanup();
   }
 };
 
 class VerifyInvokeHTTPOKResponse : public VerifyInvokeHTTP {
-public:
-  virtual void runAssertions() override {
-    assert(LogTestController::getInstance().contains("key:invokehttp.status.code value:201"));
-    assert(LogTestController::getInstance().contains("response code 201"));
+ public:
+  void runAssertions() override {
+    using org::apache::nifi::minifi::utils::verifyLogLinePresenceInPollTime;
+    assert(verifyLogLinePresenceInPollTime(std::chrono::seconds(6),
```

Comment:
This is a maximum value, and the test would probably run for far less time. `contains` was running for 3 seconds, but the `waitForVerifyProcessor` added an extra 6 seconds (line 35 of this file). I went with keeping the flat 6 seconds, because this is seems like the time frame these logs should have been produced, and not the 3 seconds the log checking introduced.